test/common_cuda.py

```diff
@@ -2,12 +2,13 @@
 
 import torch
 import torch.cuda
+from common import TEST_WITH_ROCM
 
 
 TEST_CUDA = torch.cuda.is_available()
 TEST_MULTIGPU = TEST_CUDA and torch.cuda.device_count() >= 2
 CUDA_DEVICE = TEST_CUDA and torch.device("cuda:0")
-TEST_CUDNN = TEST_CUDA and torch.backends.cudnn.is_acceptable(torch.tensor(1., device=CUDA_DEVICE))
+TEST_CUDNN = TEST_CUDA and (TEST_WITH_ROCM or torch.backends.cudnn.is_acceptable(torch.tensor(1., device=CUDA_DEVICE)))
```

Comment:
nit: It would be good to mention that `TEST_CUDNN` is code for `TEST_MIOPEN` in the ROCm build